drake/common/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@
 # Drake common library
 #
 
+drake_find_package(Protobuf REQUIRED)
+include_directories(${Protobuf_INCLUDE_DIRS})
+include_directories(${PROJECT_BINARY_DIR}/..)  # to find generated protobuf headers
```

Comment:
@jamiesnape -- i wonder if you could take a look at this for me?  it works on my system and half the build servers, but is not robust enough for the entire build matrix.